perllib/HighwaysEngland.pm

```diff
@@ -0,0 +1,35 @@
+package HighwaysEngland;
+
+use strict;
```

Comment:
Pedantically, needs a use FixMyStreet; use DBI; use Utils; as all are used by it.
(I guess ideally it wouldn't have FMS, something else would pass it in as a config, but meh, not needed now!)